samples/dfu/nrf9160/boards/nrf9160dk_nrf9160_ns.conf

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) 2022 INTELLINIUM <giuliano.franchetto@intellinium.com>
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Logging
+CONFIG_CONSOLE=y
+CONFIG_LOG=y
+
+# Download target
+CONFIG_DOWNLOAD_HOST="0.0.0.0"
+CONFIG_DOWNLOAD_FILE="app_update_uart.bin"
+#CONFIG_DOWNLOAD_FILE="app_update_uart_wrong_target.bin"
+#CONFIG_DOWNLOAD_FILE="app_update_uart_wrong_magic.bin"
```

Comment:
These should be generated and used as part of a test which can run using twister, not in here as commented lines